config/crd/bases/chaos-mesh.org_schedules.yaml

```diff
@@ -1898,6 +1898,7 @@ spec:
               description: 'TODO: use a custom type, as `TemplateType` contains other
                 possible values'
               type: string
+            workflow: {}
```

Comment:
I think this is the reason that makes the CRD invalid.

Reply:
I separate schedule totally from workflow. It no longer use embeded chaos any more, but generate its own types. Does it work now?